Cargo.toml

```diff
@@ -24,7 +24,8 @@ members = [
     "verifiers/ultraplonk", 
     "verifiers/proofofsql",
     "utils/native-cache",
-    "utils/generate-bags/node-runtime",
+    # Uncomment when in need to re-generate bags thresholds
+    # "utils/generate-bags/node-runtime",
```

Comment:
I don't like this comment and also is not necessary: even without uncomment it you can go in `utils/generate-bags/node-runtime` and run `cargo build` (I guess you need just replace the workspace references in `Cargo.toml`).

Anyway is just a mater of taste.

Reply:
You'd need to update the Cargo.toml removing all references to workspace. Doable but then you need to duplicate dependencies and you need to keep them up-to-date. Probably better to just uncomment the line in the root Cargo.toml if needed as to avoid possible headaches/inconsistencies, if you agree